### ISSUE TITLE
#-163666144------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#495]
<strong>Source:</strong>`<a id="edit-button-3b290a3d8914459ca5f67f35bbf25fe1" href="/layers/3b290a3d8914459ca5f67f35bbf25fe1/edit" aria-describedby="calcite-tooltip-8ef831cc-29d3-4bec-3e2d-64e153e38a12">`
<strong>Element Location: </strong>#edit-button-3b290a3d8914459ca5f67f35bbf25fe1
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@4cb773c5
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
